templates/mariadb_schema.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: database.openstack.org/v1beta1
+kind: MariaDBSchema
+metadata:
+  name: {{.SchemaName}}
+  labels:
+    dbName: {{.DatabaseHostname}}
```

Comment:
I haven't looked hard at MariaDB&lt;Schema&gt; but is this really how we're specifying the database server?